Cargo.lock

```diff
@@ -5346,7 +5346,7 @@ dependencies = [
  "chrono",
  "lazy_static",
  "matchers",
- "parking_lot 0.11.0",
+ "parking_lot 0.9.0",
```

Comment:
Hm, this seems surprising - maybe a mistake?

Reply:
I thought so as well, but it wasn't a change I made by hand (came from x.py check)

Reply:
hm, can you try reverting it by hand perhaps? if we can't get cargo to cooperate I'd like to page in some cargo folks before landing this

Reply:
done, :crossed_fingers: